src/main/java/com/pusher/client/channel/PrivateEncryptedChannelEventListener.java

```diff
@@ -0,0 +1,18 @@
+package com.pusher.client.channel;
+
+/**
+ * Interface to listen to private encrypted channel events.
+ */
+public interface PrivateEncryptedChannelEventListener extends ChannelEventListener {
+    /**
+     * Called when an attempt to authenticate a private encrypted channel fails.
+     *
+     * @param message
+     *            A description of the problem.
+     * @param e
+     *            An associated exception, if available.
```

Comment:
```suggestion
     * @param message A description of the problem.
     * @param e An associated exception, if available.
```
I did not see that on separate lines in other places and it reads better when it takes less real-estate.

Reply:
👍 I think this goes back to the previous point of not leaving the world worse. The `PrivateChannelEventListener` is on separate lines like this.